addons/website/static/src/builder/plugins/options/replace_media_option.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+<t t-name="website.ReplaceMediaOption" t-inherit="html_builder.ReplaceMediaOption">
+    <xpath expr="//BuilderUrlPicker" position="replace">
+        <WebsiteUrlPicker
+            title.translate="Your URL"
+            action="'setUrl'"
+            inputClasses="'o_we_large'"
+            placeholder.translate="e.g. /my-awesome-page"/>
+    </xpath>
+</t>
+
```

Comment:
For me, we don't need WebsiteUrlPicker. We should remove WebsiteUrlPicker everywhere and replace it by BuilderUrlPicker + patch BuilderUrlPicker. When website is install, each BuilderUrlPicker should propose the list.

Reply:
You should do something like patch(LinkPopover.prototype,  in "community/addons/website/static/src/js/editor/html_editor.js"
We shouldn't use autocompleteWithPages, it s legacy code.